src/App.jsx

```diff
@@ -1,9 +1,82 @@
+import { useState } from "react";
 import "./App.css";
 
 function App() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
```

Comment:
For the sake of this exercise, it's okay to handle the state for the form the way you did it, it's great... but also you should consider in case you have like a long-form how you can handle it. maybe you don't need to change for this, but it's good to handle the react forms with objects rather than having different states.
```suggestion
  const [formData, setFormData] = useState({
  firstName:"",
  lastName:""
  });
```